src/ProjectTemplates/test/IdentityUIPackageTest.cs

```diff
@@ -0,0 +1,190 @@
+            "Identity/lib/jquery-validation/LICENSE.md",
+            "Identity/lib/jquery-validation/dist/additional-methods.js",
+            "Identity/lib/jquery-validation/dist/additional-methods.min.js",
+            "Identity/lib/jquery-validation/dist/jquery.validate.js",
+            "Identity/lib/jquery-validation/dist/jquery.validate.min.js",
+            "Identity/lib/jquery-validation-unobtrusive/jquery.validate.unobtrusive.js",
+            "Identity/lib/jquery-validation-unobtrusive/jquery.validate.unobtrusive.min.js",
+            "Identity/lib/jquery-validation-unobtrusive/LICENSE.txt",
+        };
+
+        [Theory]
+        [MemberData(nameof(MSBuildIdentityUIPackageOptions))]
+        public async Task IdentityUIPackage_WorksWithDifferentOptions(IDictionary<string, string> packageOptions, string versionValidator, string[] expectedFiles)
+        {
+            Project = await ProjectFactory.GetOrCreateProject("identityuipackage" + string.Concat(packageOptions.Values), Output);
+            var useLocalDB = false;
+
+            var createResult = await Project.RunDotNetNewAsync("razor", auth: "Individual", useLocalDB: useLocalDB, environmentVariables: packageOptions);
+            Assert.True(0 == createResult.ExitCode, ErrorMessages.GetFailedProcessMessage("create/restore", Project, createResult));
+
+            var projectFileContents = ReadFile(Project.TemplateOutputDir, $"{Project.ProjectName}.csproj");
+            Assert.Contains(".db", projectFileContents);
+
+            var publishResult = await Project.RunDotNetPublishAsync(packageOptions: packageOptions);
+            Assert.True(0 == publishResult.ExitCode, ErrorMessages.GetFailedProcessMessage("publish", Project, publishResult));
+
+            // Run dotnet build after publish. The reason is that one uses Config = Debug and the other uses Config = Release
+            // The output from publish will go into bin/Release/netcoreapp3.0/publish and won't be affected by calling build
+            // later, while the opposite is not true.
+
+            var buildResult = await Project.RunDotNetBuildAsync(packageOptions: packageOptions);
+            Assert.True(0 == buildResult.ExitCode, ErrorMessages.GetFailedProcessMessage("build", Project, buildResult));
+
+            var migrationsResult = await Project.RunDotNetEfCreateMigrationAsync("razorpages");
+            Assert.True(0 == migrationsResult.ExitCode, ErrorMessages.GetFailedProcessMessage("run EF migrations", Project, migrationsResult));
+            Project.AssertEmptyMigration("razorpages");
+
+            using (var aspNetProcess = Project.StartBuiltProjectAsync())
+            {
+                Assert.False(
+                    aspNetProcess.Process.HasExited,
+                    ErrorMessages.GetFailedProcessMessageOrEmpty("Run built project", Project, aspNetProcess.Process));
+
+                var response = await aspNetProcess.SendRequest("/Identity/lib/bootstrap/dist/css/bootstrap.css");
```

Comment:
So the test here is to confirm not just that the files ended up in the produced result, but that that result is then accessible via HTTP?

Reply:
Yes